dotnetv3/Organizations/DeleteOrganizationalUnitExample/DeleteOrganizationalUnitExample/DeleteOrganizationalUnit.cs

```diff
@@ -10,23 +10,23 @@ namespace DeleteOrganizationalUnitExample
     using Amazon.Organizations.Model;
 
     /// <summary>
-    /// Shows how to delete an existing Amazon Organizations Organization Unit.
+    /// Shows how to delete an existing AWS Organizations organization unit.
```

Comment:
Here and throughout: 
organization unit. --> organizational unit.

Reply:
Fixed.